theme/ckeditor5-link/linkform.css

```diff
@@ -4,36 +4,23 @@
  */
 
 .ck-link-form {
-	padding: var(--ck-spacing-large);
+	padding: var(--ck-spacing-standard);
+
+	& .ck-button {
+		border-width: 0;
+	}
 
 	&:focus {
 		/* https://github.com/ckeditor/ckeditor5-link/issues/90 */
 		outline: none;
 	}
 
-	& .ck-input-text {
-		/* https://github.com/ckeditor/ckeditor5-link/issues/145 */
-		width: 100%;
+	& > :not(:first-child) {
+		margin-left: var(--ck-spacing-standard);
 	}
 
-	& .ck-label {
-		margin-bottom: var(--ck-spacing-tiny);
-	}
-}
-
-.ck-link-form__actions {
-	padding-top: var(--ck-spacing-large);
-
-	& .ck-button {
-		/* "Save" and "Cancel" buttons. */
-		& + .ck-button {
-			margin-right: var(--ck-spacing-medium);
-		}
-
-		/* The "Unlink" button.*/
-		&:last-child {
-			/* https://github.com/ckeditor/ckeditor5-link/issues/145 */
-			margin-right: calc(4 * var(--ck-spacing-medium));
-		}
+	& .ck-input-text {
+		/* https://github.com/ckeditor/ckeditor5-link/issues/145 */
+		min-width: 18em;
```

Comment:
We should consider variable here to let developers easily change max-width of inputs.